src/Jupyter/Magic/EstimateMagic.cs

```diff
@@ -36,10 +38,10 @@ public async Task<ExecutionResult> RunAsync(string input, IChannel channel)
 
             var qsim = new ResourcesEstimator();
             qsim.DisableLogToConsole();
-            
+
             await symbol.Operation.RunAsync(qsim, args);
 
-            return qsim.AsDictionary().ToExecutionResult();
+            return qsim.ToExecutionResult();
```

Comment:
you're returning the simulator.... curious, what happens on clients that do not support HTML, what gets generated?
Would it be better to return the `qsim.Data`, which is a [DataTable](https://docs.microsoft.com/en-us/dotnet/api/system.data.datatable?view=netstandard-2.1) and write an encoder for `DataTable`? More generic and could be used in more places. Maybe write an `DataTable` encoder for json/text too?

Reply:
Ah, good point. I checked HTML and JSON output, but forgot text/plain...